.github/workflows/test.yml

```diff
@@ -17,118 +17,36 @@ env:
-          - os: ubuntu-18.04
-            python: "3.5"
-            toxenv: cp35
-            continue-on-error: false
-          - os: ubuntu-18.04
-            python: "3.6"
-            toxenv: cp36
-            continue-on-error: false
-          - os: ubuntu-18.04
-            python: "3.7"
-            toxenv: cp37
-            continue-on-error: false
-          - os: ubuntu-18.04
-            python: "3.8"
-            toxenv: cp38
-            continue-on-error: false
-          - os: ubuntu-18.04
-            python: "3.9"
-            toxenv: cp39
-            continue-on-error: false
-
-          - os: ubuntu-18.04
-            python: "pypy2"
-            toxenv: pp2
-            continue-on-error: false
-          - os: ubuntu-18.04
-            python: "pypy3"
-            toxenv: pp3
-            continue-on-error: false
-
-          - os: ubuntu-20.04
-            python: "2.7"
-            toxenv: cp27
-            continue-on-error: false
-          - os: ubuntu-20.04
-            python: "3.5"
-            toxenv: cp35
-            continue-on-error: false
-          - os: ubuntu-20.04
-            python: "pypy2"
-            toxenv: pp2
-            continue-on-error: false
-          - os: ubuntu-20.04
-            python: "pypy3"
-            toxenv: pp3
-            continue-on-error: false
-
-          - os: macos-10.15
-            python: "3.5"
-            toxenv: cp35
-            continue-on-error: true
-          - os: macos-11.0
-            python: "3.5"
-            toxenv: cp35
-            continue-on-error: true
-          - os: windows-2019
-            python: "3.5"
-            toxenv: cp35
-            continue-on-error: false
-
-          # 3.10 Alpha
-          - os: ubuntu-18.04
-            python: "3.10.0-alpha.2"
-            toxenv: "cp310"
-            continue-on-error: false
-          - os: macos-11.0
-            python: "3.10.0-alpha.2"
-            toxenv: "cp310"
-            continue-on-error: true
-          - os: windows-2019
-            python: "3.10.0-alpha.2"
-            toxenv: "cp310"
-            continue-on-error: true
-
+        python-version: [3.7, 3.8, 3.9, "3.10", "3.11", '3.12', 'pypy2', 'pypy3']
+        os: [windows-2019, ubuntu-20.4, macos-11]
+    runs-on: ${{ matrix.os }}  
     steps:
-      - uses: actions/checkout@v2
-      - name: Setup python
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python }}
-      - name: Install
-        run: pip install -q -r "tox-requirements.txt"
-      - name: Run tests
-        continue-on-error: ${{ matrix.continue-on-error }}
-        env:
-          TOXENV: ${{ matrix.toxenv }}
-        run: tox -v
-
+    - uses: actions/checkout@v4
+    - name: Setup python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python }}
+    - name: Install
+      run: |
+        pip install -e .
+        pip install -r "test-requirements.txt"
```

Comment:
Looks like the requirements file is named `test_requirements`

Reply:
bad copy-pasta. Sorry for the noise